### DISCUSSION TITLE
Error with Missing Foreground with MVTec.py

### DISCUSSION THREAD

**Discussion by yxteo2** [cqylunlun/GLASS#36] [Q&A] [answered]
I adjusted the file path in run-mvtec.sh, but I got this error.
The problem is other fg_mask file is able to load but only this one, so i got no idea where gone wrong

`FileNotFoundError: [Errno 2] No such file or directory: 'MVTec/fg_mask/carpet/088.png'`

**Comment by cqylunlun** [comment, marked as answer]
Please refer to issue #3 , download the [fg_mask](https://github.com/cqylunlun/GLASS?tab=readme-ov-file#4foreground-mask-download-link) folder and place it at the specified path.
